database/flyway.sh

```diff
@@ -11,4 +11,4 @@ if [ "$#" -ne 2 ]; then
 	exit 2
 fi
 
-flyway -user=$DB_USERNAME -password=$DB_PASSWORD  -url=jdbc:mysql://$DB_HOSTNAME:$DB_PORT/$DB_NAME -locations=filesystem:$PWD/migration -baselineOnMigrate=true -sqlMigrationSuffix=.sql $1
+flyway -user=$DB_USERNAME -password=$DB_PASSWORD  -url=jdbc:mysql://$DB_HOSTNAME:$DB_PORT/$DB_NAME "-locations=filesystem:$PWD/migration" -baselineOnMigrate=true -sqlMigrationSuffix=.sql $1
```

Comment:
This works if you wrap the option in there too? I'd expect to see something like `-locations=filesystem:"$PWD/migration"`.